src/feature-management/FeatureFlagsProvider.tsx

```diff
@@ -0,0 +1,50 @@
+import React, {useEffect, useState} from "react";
+import Rox from "rox-browser";
+import {flags} from "./flags.ts";
+import {FeatureFlagsContext, useFeatureFlags} from "./index.ts";
+
+// TODO: insert your SDK Key from https://cloudbees.io/ here.
+const sdkKey = '<INSERT YOUR SDK KEY HERE>'
+
+type Props = {
+  children?: React.ReactNode
+};
+
+export const FeatureFlagsProvider = ({children} : Props): React.ReactNode => {
+
+
+  const [flagState, setFlagState] = useState(useFeatureFlags())
```

Comment:
This looks weird, to use a hook (`useFeatureFlags `) to initialise another hook (`useState `)